### DISCUSSION TITLE
⭐ Important points to remember when submitting your project! ⭐

### DISCUSSION THREAD

**Discussion by aycabas** [microsoft/hack-together-teams#96] [General]
Amazing hackathon participants, we are so excited to see your projects coming together and celebrate the end of the hackathon in the [live session](https://aka.ms/hack-together-teams/session6) today! 🥳

Couple of points I'd like to remind:
- Make sure to **[register](https://aka.ms/hack-together-teams/register) to the hackathon if you are submitting your project**, so you'll be eligible for the digital badge and other prizes.
- If you are submitting your project as a team, **all team members should [register](https://aka.ms/hack-together-teams/register) to the hackathon separately** so all team members will be eligible for the digital badge and other prizes.
- When submitting your project, **providing a public repository of your project together with a short demo video is mandatory**, so our judges will be able to review your project submission.
- Don't forget to **[submit your project](https://github.com/microsoft/hack-together-teams/issues/new?assignees=&labels=&projects=&template=project.yml&title=Project%3A+%3Cshort+description%3E) with all the required information filled in until June 15th, 11:59 PM PST**.

>Anyone who submits their Teams app to the hackathon will receive a digital badge and be eligible for [the other prizes](https://github.com/microsoft/hack-together-teams#judging-and-prizes-)! We are so excited to finally see what you've build and celebrate your achievements! 💜

**Comment by mookmac** [comment]
hi Ayca, do you have any suggestions for software to use to record a suitable video?

**Reply by aycabas** [reply]
Hi @mookmac, you can you Snipping Tool to record your screen if you are using Windows. That's the easiest way and free. If you are using Mac I'm not sure what's the easiest, @waldekmastykarz might have some suggestions there 🙂

**Reply by svrooij** [reply]
@mookmac You can also use the Xbox Game Bar, once you start it you go to the windows you wish to record and press Win+alt+r, and repeat to stop recording. It also does audio, but they seem to promoting it only for games.

**Reply by mookmac** [reply]
thanks both, I'm on Windows 10 so I don't get the video recording option in Snipping Tool. Xbox Game Bar works really well though, appreciate the tip!
